src/text/parsers/timestamp.rs

```diff
@@ -454,3 +456,9 @@ mod reader_tests {
         Ok(())
     }
 }
+
+/// Matches the next input character if it is a base-10 digit. This wraps [char::is_digit] in the
+/// nom parsing function signature.
+pub(crate) fn digit(input: &str) -> IResult<&str, char> {
```

Comment:
🗺️ This low-level parser is used with `nom` parsers so widely in the `timestamp` module that I decided it wasn't worth it to convert its return type to `IonParseResult`. (It is not new; it used to live in `numeric_support.rs` but I realized it only used in `text/parsers/timestamp`).